src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -2136,6 +2136,10 @@ static double accSymId2alter(SymId id)
         break;
     case SymId::accidentalDoubleSharp:                     res =  2;
         break;
+    case SymId::accidentalSori:                            res =  0.5;
```

Comment:
In accidental.cpp the offsets are 0.33 and -0.67, respectively, changed to that via 56e76ba after I found some evidence for that, see https://de.wikipedia.org/wiki/Dastgah#Die_Dastg%C4%81hs_mit_ihren_%C4%80w%C4%81z:

**Erläuterung: B=H; is: um einen Halbton erhöht; b: um einen Halbton erniedrigt; p [„koron“]: um grob einen Viertelton (etwas genauer: um 2/3 eines Halbtons)[[30]](https://de.wikipedia.org/wiki/Dastgah#cite_note-30) erniedrigt; > [„sori“]: um grob einen Viertelton (genauer: um 1/3 eines Halbtons) erhöht.**

Reply:
Agreed.